docs/modules/reference/pages/glossary.adoc

```diff
@@ -28,35 +31,55 @@ There, users can find knowledge base articles, support, announcements, and gener
 
-Flows:: Summaries of network traffic sent by network devices. 
+Events:: Structured historical records of activities in {page-component-title} and the nodes, interfaces, and services it manages.
+(See xref:operation:events/anatomy-events.adoc[Anatomy of an Event].)
+
+Flows:: Summaries of network traffic sent by network devices.
 (See xref:operation:flows/introduction.adoc[Flows].)
 
 Foreign Source:: A data source hosted outside of your local network.
 To customize monitoring of nodes in a foreign source, you can set a foreign source definition with your desired detectors and policies.
-(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign source definition].)
+(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign Source Definition].)
+
+Geolocation:: The actual physical location of a node monitored by {page-component-title}.
+Nodes can be manually assigned a location, or the location can be automatically determined using the xref:reference:provisioning/adapters/geoip.adoc[GeoIP Provisioning Adapter].
 
-Helm:: Helm is an an OpenNMS application for creating flexible dashboards to display and customize fault management and performance management data from Horizon and Meridian.
+Grafana:: Grafana is an open-source analysis and visualization web application.
+It connects to data sources and generates dashboards with charts, graphs, and alerts.
+(See https://grafana.com/[Grafana].)
+OpenNMS HELM is a Grafana plugin that allows users to create customized dashboards with data from {page-component-title}.
+(See https://docs.opennms.com/helm/latest[HELM Documentation].)
+
+Helm:: Helm is an OpenNMS application for creating flexible dashboards to display and customize fault, flow, and performance data from {page-component-title}.
 Helm works with the Grafana analytics program to customize the data you can view.
 Not to be confused with the https://helm.sh/[Kubernetes Helm] package manager.
-(See https://docs.opennms.com/helm/latest[HELM documentation].)
+(See https://docs.opennms.com/helm/latest[HELM Documentation].)
 
 Horizon:: Open-source solution from OpenNMS that helps users visualize and monitor everything on their local and remote networks.
 Free, community-driven project includes the latest technology and features delivered through a rapid release cycle.
 
-IFTTT:: If This Then That (https://ifttt.com/[IFTTT]) is an automation and workflow service that integrates apps, devices, and services. 
+IFTTT:: If This Then That (https://ifttt.com/[IFTTT]) is an automation and workflow service that integrates apps, devices, and services.
 {page-component-title} can integrate with IFTTT to trigger actions on other services based on customized alarms.
 (See xref:operation:alarms/ifttt-integration.adoc[IFTTT Integration].)
 
+Instance:: (1) A single, unique occurrence of an element, document, or running program.
+(2) Used in data collection for storing tabular data as separate data points.
```

Comment:
Bonnie's last comment on the previous thread was hidden, since it was marked Resolved when I committed the change, but does this work to accommodate it?

```suggestion
Instance:: (1) A single, unique occurrence of an element, document, or running program.
(2) Used in data collection for storing tabular data as separate data points.
(3) Used to refer to the OpenNMS Core in the OpenNMS Appliance Service UI.
```

Reply:
Should it be `OpenNMS Core` or `{page-component-title} Core`?

Reply:
Good catch. {page-component-title} core

Reply:
Got it! If we're all good with this version, I'll commit it. :) 

```suggestion
Instance:: (1) A single, unique occurrence of an element, document, or running program.
(2) Used in data collection for storing tabular data as separate data points.
(3) Used to refer to the {page-component-title} Core in the OpenNMS Appliance Service UI.
```